components/vf-sass-config/mixins/_blockquote.scss

```diff
@@ -1,3 +1,6 @@
+// Reusable styling for html blockquote elements
+// @include blockquote;
+
 @mixin blockquote() {
   @include set-type(text-body--2);
   margin: 0;
```

Comment:
could we include the Sass variable in the `set-type` mixin instead of `margin: 0` here?

I know it's slightly out of the PR's remit

Reply:
Took a stab at this. Further in the weeds: I have an itch that the blockquote mixin should live within the `vf-blockquote` component, but that would make various Sass import difficulties 🙈

Reply:
Yeah, and if we did it for `vf-blockquote` we could be opening up a little can of worms –– "should the button mixin be with buttons‽" etc etc.

I like the separation as it gives the opportunity for people to use `vf-core` as a little mixin library, like you can do with Bootstrap.